packages/adblocker/src/filters/network.ts

```diff
@@ -400,6 +400,20 @@ function compileRegex(
   return new RegExp(filter);
 }
 
+export function findLastIndexOfUnescapedCharacter(text: string, character: string) {
+  let lastIndex = text.lastIndexOf(character);
+
+  if (lastIndex === -1) {
+    return -1;
+  }
+
+  while (lastIndex >= 0 && text.charCodeAt(lastIndex - 1) === 92 /* '\\' */) {
```

Comment:
Nit. If `lastIndex` is 0 then we probably want to exit the loop.
```suggestion
  while (lastIndex > 0 && text.charCodeAt(lastIndex - 1) === 92 /* '\\' */) {
```